search_gov_crawler/elasticsearch/i14y_helper.py

```diff
@@ -20,18 +26,39 @@
 }
 # fmt: on
 
-def null_date(article_date):
+load_dotenv()
+logging.basicConfig(level=os.environ.get("SCRAPY_LOG_LEVEL", "INFO"))
+logging.getLogger().handlers[0].setFormatter(JsonFormatter(fmt=LOG_FMT))
+logger = logging.getLogger("search_gov_crawler.i14y_helper")
+
+
+def parse_date_safely(date_value: Any) -> str | None:
     """
     Convert falsey date values (like an empty string) to None,
-    which will yield a null value in elasticsearch
+    which will yield a null value in elasticsearch.
 
     Args:
-        article_date (str | Any): Date value that needs to fallback to null
+        date_value (str | Any): Date value that needs to fallback to null
 
     Returns:
-        str: Either the original date, or NoneValue
+        str: Either the date in isoformat (YYYY-MM-DD'T'HH:MM:SS), or None
     """
-    return article_date or None
+    datetime_format = "%Y-%m-%dT%H:%M:%S"
+
+    if date_value is None or date_value == "":
+        return None
+
+    if isinstance(date_value, datetime):
+        return date_value.strftime(datetime_format)
+
+    try:
+        datetime_object = parser.parse(date_value, fuzzy=True)
```

Comment:
We should ignore timezone or it will fail on dates like: `January 8, 2013 10:05:30 AM EST`
```suggestion
        datetime_object = parser.parse(date_value, ignoretz=True, fuzzy=True)
```
I think it's safer to assume that all timestamps are in UTC, or at least treat them as such. In some edge cases the worst that would happen is that we would be off by some hours (or one day)

Reply:
if you pass a naive date string to elasticsearch, what timezone does it think its in?

Reply:
It sees it as UTC, and it is convention to store timestamps as UTC. I guess in this case, if we are supporting timezones, we should be sure to apply the offset to get the clean/pure UTC, something like `datetime_object.astimezone(timezone.utc)`